### ISSUE TITLE
Go 1.10: reference_test.go:311: Errorf format %v arg r1.IsCanonical is a func value, not called

### ISSUE THREAD

**Issue by nim-nim** [go-openapi/jsonreference#2]
36d33bfe519efae5632669801b180bf1a245da3b does not pass unit tests with Go 1.10:

```
+ GOPATH=/builddir/build/BUILD/jsonreference-36d33bfe519efae5632669801b180bf1a245da3b/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro  '\'''
# github.com/go-openapi/jsonreference
./reference_test.go:311: Errorf format %v arg r1.IsCanonical is a func value, not called
FAIL    github.com/go-openapi/jsonreference [build failed]
```

**Comment by casualjim** [comment]
fixed: 524857335d7f7f8ba1dc32bd3fe427b4b49fbcaa

**Comment by nim-nim** [comment]
Thank you !
